src/Table/Footer.tsx

```diff
@@ -60,15 +61,27 @@ export default function Footer({
     return null;
   }
 
-  // Find the widest possible number range <= `VISIBLE_PAGE_RANGE_SIZE`.
-  const pageNumberPadding = (VISIBLE_PAGE_RANGE_SIZE - 1) / 2;
+  // As a safeguard for when the typescript PageSizeNumber type is not enforced (jsx files),
+  // check if the visiblePageRangeSize is not a PageSizeNumber (even, < 1, > 11),
+  // and if so, set it to the default value.
+  let safeVisiblePageRangeSize = visiblePageRangeSize;
+  if (
+    safeVisiblePageRangeSize < 1 ||
+    safeVisiblePageRangeSize > 11 ||
+    safeVisiblePageRangeSize % 2 === 0
```

Comment:
Truly the most minor of nits: Let's use `visiblePageRangeSize`  in the comparison checks here since the the page range size is only safe once we've done this check and possibly performed the assignment below.

```ts
let safeVisiblePageRangeSize = visiblePageRangeSize;
if (
  visiblePageRangeSize < 1 ||
  visiblePageRangeSize > 11 ||
  visiblePageRangeSize % 2 === 0
) {
  safeVisiblePageRangeSize = DEFAULT_VISIBLE_PAGE_RANGE_SIZE;
}
```